plugins/vst_base/VstPlugin.cpp

```diff
@@ -66,27 +69,93 @@
 #	include <X11/Xlib.h>
 #endif
 
+namespace PE
+{
+// Utilities for reading PE file machine type
+// See specification at https://msdn.microsoft.com/library/windows/desktop/ms680547(v=vs.85).aspx
+
+// Work around name conflict
+#ifdef i386
+#	undef i386
+#endif
+
+enum class MachineType : uint16_t
+{
+	unknown = 0x0,
+	amd64 = 0x8664,
+	i386 = 0x14c,
+};
+
+class FileInfo
+{
+public:
+	FileInfo(QString filePath)
+		: m_file(filePath)
+	{
+		m_file.open(QFile::ReadOnly);
```

Comment:
This file open can fail if the path passed to `VstPlugin` is relative. So I can't load some VSTs...
There should be a path check logic like one in `VstPlugin::tryLoad`.

Reply:
Fixed via 7ddca851d1b18bbc64b7d1ee15078b0daa8501c9